.github/workflows/verify.yml

```diff
@@ -45,6 +45,25 @@ jobs:
     - name: Install dependencies (Nim)
       run: sudo apt install nim
 
+    - name: Set up Rust (1.42.0)
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: 1.42.0-x86_64-unknown-linux-gnu
+        default: true
+        profile: minimal
+
+    - name: Set up Rust (nightly)
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: nightly-x86_64-unknown-linux-gnu
+        default: true
+        profile: minimal
+
```

Comment:
同時に必要です。

後者はcargo-udepsに必要です。

前者についてですが、Rustは1.5ヶ月に一度のリリースのたびに新機能や新APIが追加されまくります。一方コンテストサイトはそんな頻度でコンパイラを更新しないので、使えない機能を使っていないことを保証する必要があります。

[現在の各プラットフォームで使えるRust](https://github.com/rust-lang-ja/atcoder-rust-resources/wiki/Since-1.15.1#%E7%8F%BE%E5%9C%A8%E3%81%AE%E5%90%84%E3%83%97%E3%83%A9%E3%83%83%E3%83%88%E3%83%95%E3%82%A9%E3%83%BC%E3%83%A0%E3%81%A7%E4%BD%BF%E3%81%88%E3%82%8Brust)

(ちなみにactions-rs/toolchainの実行時間は、GitHub Actionsの調子が良いときは10sくらいで悪いときは20sくらいです。体感では。)

Reply:
同時に必要、了解です。